helm/app/templates/dashboard-role.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: {{ include "app.fullname" $ }}
+  {{- if hasKey $values "namespace" }}
+  namespace: {{ $values.namespace }}
+  {{- end }}
+  labels:
+    helm.sh/chart: {{ include "app.chart" $ }}
+    app.kubernetes.io/name: {{ include "app.name" $ }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+rules:
+- apiGroups:
+  - "*"
+  resources:
+  - "*"
+  verbs:
+  - get
+  - list
+  - watch
+  - describe
+  - exec
+  - update
+  - delete
+- apiGroups:
+  - ""
+  resourceNames:
+  - kubernetes-dashboard-csrf
```

Comment:
isnt it this one now? 
https://github.com/vardius/go-api-boilerplate/pull/49/files#diff-70fe989fcc5a09896610bf02a0517a31R267

Reply:
Yes, fixed.